## Velodyne model
sensor        : velodyne16

## Velodyne FoV
# horizontal/azimuth
max_ang_w     : 80
# vertical/altitude
max_ang_h     : 45

## Velodyne angular resolution

# The following aren't for a pretty image
# but accurate reconstructed pcl

# horizontal/azimuth resolution
#ang_res_x     : 0.1
# vertical/altitude resolution
#ang_res_y     : 2

# The following are for pretty image
# but inaccurate pcl

# horizontal/azimuth resolution
ang_res_x     : 0.25
# vertical/altitude resolution
ang_res_y     : 0.35

## Velodyne min/max range
min_range     : 0.4
max_range     : 10

# pointcloud frame type
# see pcl RangeImage
laser_frame   : False

# publish an rgb image
rgb_range_img : False
